### ISSUE TITLE
PCA analysis doesn't look correct after upgrading

### ISSUE THREAD

**Issue by aopisco** [scverse/scanpy#991]
![image (10)](https://user-images.githubusercontent.com/25906087/72300922-09d19080-361a-11ea-8834-0cdc35b7b367.png)

The PCs are no longer in order after upgrading to v.1.4.5.post1

**Comment by gokceneraslan** [comment]
Just to clarify, are you referring to 3 plots in the middle (PCA loading plots)? In new scanpy release, we render both positive and negative genes.

**Comment by aopisco** [comment]
no, the variance ratio, first PC is PC21 and the last two are PC50 and PC20

**Comment by ivirshup** [comment]
@aopisco, I'm having some trouble reproducing your plots. Could you show us some more details (software versions mainly) about your environment?

**Comment by Zethson** [comment]
We'd need a reproducer to have a look. Closing for now.
